src/app/projects/projects.html

```diff
@@ -1,5 +1,82 @@
 <section layout="column" flex>
-  <navbar projects="$projects.projects" user="gapi.user"></navbar>
+  <navbar projects="projects.map" user="gapi.user"></navbar>
+  <div ng-if="ACTIVE_STATE.name === 'projects'">
```

Comment:
Would you mind leading that charge? I find this stuff quite complicated, and I never seem to strike the right solution. It'd probably just be a time waster for all of us to spend my time there :)